src/app/common/hooks/use-explorer-link.ts

```diff
@@ -10,13 +10,13 @@ import { openInNewTab } from '../utils/open-in-new-tab';
 export interface HandleOpenTxLinkArgs {
   blockchain: Blockchains;
   suffix?: string;
-  txId: string;
+  txid: string;
```

Comment:
I think as in [this PR](https://github.com/leather-wallet/extension/pull/4488) I had to make a change that broke the explorer link. 

It's not a big deal to me what convention to use but sometimes for stuff like this it's easier to just use what the payload provides

Reply:
[Slack discussion](https://trustmachines.slack.com/archives/C05LAP952E7/p1699351436402149). Let's just be consistent. The _txid_ is so common in crypto that we can consider it a single thing. We're not making it up, txid is used everywhere.

I'm generally against too many abbreviations/contractions. Taproot > Tr etc. But some exceptions can be made. E.g. there's no doubt what "tx" is. Likewise, a txid is a single concept. We actually say the letters when talking about it, rather than "transaction id".